LICENSE.md

```diff
@@ -0,0 +1,5 @@
+## CC-BY 4.0 License
+____
+Copyright (c) 2016-2019 Uphold
+
+This work is licensed under the Creative Commons Attribution 4.0 International License. To view a copy of this license, visit http://creativecommons.org/licenses/by/4.0/ or send a letter to Creative Commons, PO Box 1866, Mountain View, CA 94042, USA.
```

Comment:
While you're at it, please add line breaks after the period, and after the link, to avoid having an excessively long line.